game/domain/view/mapview.lua

```diff
@@ -36,24 +36,28 @@ function MapView:draw()
   for i = 0, map.h-1 do
     for j = 0, map.w-1 do
       local tile = map.tiles[i+1][j+1]
-      local body = map.bodies[i+1][j+1]
-      local x, y = j*TILE_W, i*TILE_H
-      g.push()
-      g.translate(x, y)
-      g.setColor(tile)
-      g.rectangle("fill", 0, 0, TILE_W, TILE_H)
-      if body then
+      if tile then
+        local body = map.bodies[i+1][j+1]
+        local x, y = j*TILE_W, i*TILE_H
         g.push()
-        g.translate(TILE_W/2, TILE_H/2)
-        g.scale(TILE_W, TILE_H)
-        g.setColor(200, 100, 100)
-        g.polygon('fill', 0.0, -0.8, -0.25, 0.0, 0.0, 0.2)
-        g.setColor(90, 140, 140)
-        g.polygon('fill', 0.0, -0.8, 0.25, 0.0, 0.0, 0.2)
+        g.translate(x, y)
+        g.setColor(tile)
```

Comment:
I'm not sure about this representation of tiles. Obviously it will change later on.